shop/public/template-cdn.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html lang="en" dir="ltr">
+
+<head>
+  <title>TITLE</title>
+  <meta charset="utf-8">
+  <meta name="viewport"
+    content="width=device-width, initial-scale=1, maximum-scale=1, shrink-to-fit=no, user-scalable=no">
+  <meta http-equiv="Content-Language" content="en">
+  <meta name="description" content="META_DESC">
+
+  <% if (htmlWebpackPlugin.options.sentryDSN) { %>
+  <script src="https://browser.sentry-cdn.com/5.19.2/bundle.min.js"
+    integrity="sha384-JpEWxJ7oLRn+baXBgcCKEv73uYWsgouzEResgJneOXvTjDZ+1crAXcNAKZoiL96Z"
+    crossorigin="<%= webpackConfig.output.crossOriginLoading %>"></script>
+  <script>
+    <% /* To listen on react */ %>
+    window._sentryLoaded = true
+
+    Sentry.init({
+      dsn: '<%= htmlWebpackPlugin.options.sentryDSN %>',
+      environment: '<%= htmlWebpackPlugin.options.sentryEnvironment %>',
+    })
+  </script>
+  <% } %>
+
+  <script>
+    localStorage.ognNetwork = "NETWORK"
+    window.ognNetwork = "NETWORK"
+  </script>
+
+  <link rel="data-dir" href="DATA_DIR" />
+  <link rel="icon" href="FAVICON" />
+</head>
+
+<body>
+  <div id="app">
+    <div class="loading-screen"></div>
+  </div>
+</body>
+<script crossorigin src="dist/openpgp.min.js"></script>
```

Comment:
Could point these at ui.ogn.app/ui.staging.ogn.app, as well

Reply:
I tried that originally but web workers can't work cross-domain

Reply:
Could you elaborate?  Not really familiar.  Is web workers functionality that openpgp uses, I guess?

Reply:
Yea it's just for the openpgp webworker stuff, which it relies on. I haven't looked into it much yet, there might be a workaround, but this was the quickest fix for now